src/main/java/link/infra/indium/renderer/render/BaseFallbackConsumer.java

```diff
@@ -82,28 +80,17 @@ public void accept(BakedModel model) {
 		final Value defaultMaterial = blockInfo.defaultAo && model.useAmbientOcclusion() ? MATERIAL_SHADED : MATERIAL_FLAT;
 		final BlockState blockState = blockInfo.blockState;
 
-		for (int i = 0; i < 6; i++) {
-			final Direction face = ModelHelper.faceFromIndex(i);
-			final List<BakedQuad> quads = model.getQuads(blockState, face, random.get());
+		for (int i = 0; i <= ModelHelper.NULL_FACE_ID; i++) {
+			final Direction cullFace = ModelHelper.faceFromIndex(i);
+			final List<BakedQuad> quads = model.getQuads(blockState, cullFace, random.get());
 			final int count = quads.size();
 
 			if (count != 0) {
-				for (int j = 0; j < count; j++) {
-					final BakedQuad q = quads.get(j);
-					renderQuad(q, face, defaultMaterial);
+				for (final BakedQuad q : quads) {
```

Comment:
Revert the change to an enhanced for loop; this is inconsistent with Indigo and less efficient.